pytential/symbolic/execution.py

```diff
@@ -210,7 +210,7 @@ def map_node_coordinate_component(self, expr):
         if isinstance(x, DOFArray):
             return thaw(self.array_context, x)
         else:
-            return self.array_context.thaw(x)
+            return self.array_context.thaw(x).copy()
```

Comment:
Why is that copy here?

Reply:
This case happens for `PointsTarget` and `PointPotentialSource` type things, which have stacked nodes of some shape ``(dim, nnodes)``. 

From what I can tell, the issue is a missing `default_offset=lp.auto` in [sumpy.p2p](https://github.com/inducer/sumpy/blob/b0ec769453fb8052da1cbd5a8ab6070e6015de52/sumpy/p2p.py#L216-L242), but I tried adding that and it had no effect. Any idea why that would be?

Reply:
> I tried adding that and it had no effect. Any idea why that would be?

Did you clear/ignore caches after you added it? `SUMPY_NO_CACHE=1`.

Reply:
Yeah.. :facepalm: 

inducer/sumpy#41